src/commands/project.lisp

```diff
@@ -282,7 +282,7 @@
          "Project: "
          :completion-function (lambda (x) (completion-strings x candidates))
          :test-function (lambda (name) (member name candidates :test #'string=)))
-        (show-message "No projects." :timeout 5))))
+        (progn (show-message "No projects." :timeout 5) nil))))
```

Comment:
How about writing `(editor-error "No projects.")` ?

Reply:
Thanks, I just made the change